lib/generators/boring/rswag/install/install_generator.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+module Boring
+  module Rswag
+    class InstallGenerator < Rails::Generators::Base
+      DEFAULT_RAILS_PORT = '3000'
+
+      desc "Adds rswag gem to the application"
+
+      class_option :rails_port,
+                   type: :string,
+                   desc: "Tell us the port number where you normally run your rails app. Defaults to PORT #{DEFAULT_RAILS_PORT}",
+                   default: DEFAULT_RAILS_PORT
+
+      def verify_presence_of_rspec_gem
+        gem_file_content_array = File.readlines("Gemfile")
+
+        rspec_is_installed = gem_file_content_array.any? { |line| line.include?("rspec-rails") }
+
+        return if rspec_is_installed
+
+        say "We couldn't find rspec-rails gem which is a dependency for the rswag gem. Please configure rspec and run the generator again!"
+
+        abort
+      end
+
+      def add_rswag_gem
+        say "Adding rswag gem", :green
+
+        run "bundle add rswag"
+      end
+
+      def run_rswag_install_generator
+        say "\nRunning rswag install generator to add required files", :green
+
+        run "bundle exec rails generate rswag:install"
+      end
+
+      def update_api_host
+        say "\nUpdating API Host URL"
+
+        rails_port = options[:rails_port]
+
+        gsub_file "spec/swagger_helper.rb",
+                  "url: 'https://{defaultHost}'",
+                  "url: 'http://{defaultHost}'"
+        gsub_file "spec/swagger_helper.rb",
+                  "default: 'www.example.com'",
+                  "default: 'localhost:#{rails_port}'"
+      end
+
+      def show_whats_next_message
+        say "\nNow that you have rswag installed, what's next? You could do the following:", :green
+
+        help_text = <<~TEXT
+          1. Write a rswag spec by following instructions from Getting Started guide at https://github.com/rswag/rswag#getting-started
+          2. Once the spec is ready, generate the swagger file with `rails rswag`
+          3. Run the rails server with `rails s`
+          4. Assuming your app is at "localhost:3000", you will find your API docs at `http://localhost:3000/api-docs/index.html`
+        TEXT
+
+        say "\n#{help_text}", :yellow
+      end
+
+      def show_additional_configurations_message
+        say "\nYou are now ready to develop your API documentation locally but for making it production ready, you might need to add more configurations. Following guides can help you:", :green
+
+        help_text = <<~TEXT
```

Comment:
I think we should automate this flow with ideal approach you think is right here. We can give them option to skip these steps if they don't agree with the generator defaults. 

The entire idea of a boring generator is that you don't have to figure out stuff on your own. The generator does that for them.